cmd/kubectl-gadget/deploy.go

```diff
@@ -35,6 +36,11 @@ func init() {
 		"traceloop", "",
 		true,
 		"enable the traceloop gadget")
+	deployCmd.PersistentFlags().StringVarP(
+		&traceloopLoglevel,
+		"traceloop-loglevel", "",
```

Comment:
What about having `info` by default to avoid the check in `entrypoint.sh`?

Reply:
The default is "info,json" in the line below. The `""` here is about the lack short version of the option (no `-t`).

Reply:
Do you want to change the default or to remove the check in `entrypoint.sh`?

Reply:
oh right, I read the wrong field. I think it's fine to keep the check that as well.